src/sentry/auth/providers/saml2/provider.py

```diff
@@ -152,6 +154,14 @@ def dispatch(self, request, helper):
 
         helper.bind_state("auth_attributes", auth.get_attributes())
 
+        # 5/28/2020 Temporarily adding logging here to check if any IdPs send a SessionNotOnOrAfter
```

Comment:
I like to format these comments as `XXX(slohmes): 5/28/2020 Temporarily ...`

Just so it's easy right away to see who did the thing. The [`XXX`](https://en.wikipedia.org/wiki/Comment_%28computer_programming%29#Tags) is a nice way to say "hey this is important to see!' (though maybe actually `NOTE` would be better here